### ISSUE TITLE
No alignment for mouse TRBV 31

### ISSUE THREAD

**Issue by CGR731** [milaboratory/mixcr#245]
Anyone else getting an error that the repseqio library cannot map reads to mTRBV31? I can see from other methods that these reads exist but mixcr ignores them

**Comment by dbolotin** [comment]
Hi,

sorry, this is a known problem, it is already fixed in our library (this [commit](https://github.com/repseqio/library/commit/26f35fda3cd0458136e11a2cf6e3d11ec843aff4)), and will be released soon.

We plan to release the library first, as soon as it will be done, I'll let you know how to use the library in a separate file.

**Comment by CGR731** [comment]
Thanks Dmitry, I look forward to the release!

**Comment by dbolotin** [comment]
I will leave the issue open until we really fix this by updating the library inside MiXCR.
Cheers.

**Comment by dbolotin** [comment]
Updated library is released. It can be downloaded from here:
https://github.com/repseqio/library/releases/download/v1.3/repseqio.v1.3.json.gz
and placed either in `~/.mixcr/libraries/` folder or in `libraries` subfolder of installation folder (`/path/to/mixcr/libraries/`). No decompression is needed, mixcr will read directly form `.json.gz` file.

You can check list of paths where mixcr searches for libraries by typing:
```
mixcr -v
```
Analysis using updated library can be done with the following command:
```
mixcr align --library repseqio.v1.3 input_R1.fastq.gz input_R2.fastq.gz output.vdjca
```
no need to specify library on the following analysis steps, MiXCR will find it automatically.

P.S. This library will be included in MiXCR 2.1.4 as default library.

**Comment by CGR731** [comment]
Works great, thanks again!
